src/main/java/com/cafe/management/system/model/response/UsersResponse.java

```diff
@@ -0,0 +1,24 @@
+package com.cafe.management.system.model.response;
+
+import com.cafe.management.system.model.entities.User;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class UsersResponse {
+
+
+    private List<User> content;
+    private int pageNo;
+    private int pageSize;
+    private long totalElements;
+    private int totalPages;
+    private boolean last;
+
```

Comment:
Use **Pageable** class for paging , request and response

ref: https://reflectoring.io/spring-boot-paging/